docs/en/latest/plugins/proxy-rewrite.md

```diff
@@ -37,6 +37,7 @@ The `proxy-rewrite` is an upstream proxy information rewriting plugin, which sup
 
 | Name      | Type          | Requirement | Default | Valid             | Description                                                  |
 | --------- | ------------- | ----------- | ------- | ----------------- | ------------------------------------------------------------ |
+| method    | string        | optional    |         | ["GET", "POST","PUT","PATCH","DELETE"] | New HTTP method.                        |
```

Comment:
The position order of the `method` fields should be consistent in both Chinese and English documents.